types/react-router/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React Router 4.0
+// Type definitions for React Router 4.1
```

Comment:
sorry got confused PR checkboxes and didn't realise that it's not definition versioning but version of original library, it's fixed now.